lib/blueprinter/version.rb

```diff
@@ -1,3 +1,3 @@
 module Blueprinter
-  VERSION = '0.8.0'
+  VERSION = '0.9.0'
```

Comment:
Would you also update the CHANGELOG.md since you're already bumping the version here? =)

Reply:
will do.